new/LIT7150Miracle.xml

```diff
@@ -0,0 +1,130 @@
+                        </desc>
+                    </relation>
+                    <relation name="lawd:hasAttestation" active="LIT7150Miracle" passive="EMML 8828">
+                        <desc>see <ref type="mss" corresp="8828"> EMML 8828, f. 20v</ref>
+                        </desc>
+                    </relation>
+                    <relation name="lawd:hasAttestation" active="LIT7150Miracle" passive="EMML4205">
+                        <desc>see <ref type="mss" corresp="EMML4205">EMML 4205, ff. 25v, 51r,
+                            26r</ref>
+                        </desc>
+                    </relation>
+                    <relation name="lawd:hasAttestation" active="LLIT7150Miracle" passive="EMML2378">
+                        <desc>
+                            <ref type="mss" corresp="EMML2378">EMML 2378, f. 31r</ref>
+                        </desc>
+                    </relation>
+                    <relation name="lawd:hasAttestation" active="LIT7150Miracle" passive="BNFabb114">
+                        <desc>
+                            <ref type="mss" corresp="BNFabb114"> BNFabb114, f. 18ra </ref>
+                        </desc>
+                    </relation>
+                    <relation name="lawd:hasAttestation" active="LIT7150Miracle" passive="EMML2059">
+                        <desc>
+                            <ref type="mss" corresp="EMML2059">EMML 2059, f. 135r</ref>
+                        </desc>
+                    </relation>
+                    <relation name="lawd:hasAttestation" active="LIT7150Miracle" passive="EMML2060">
+                        <desc>
+                            <ref type="mss" corresp="EMML2060">EMML 2060, f. 183r</ref>
+                        </desc>
+                    </relation>
+                    <relation name="lawd:hasAttestation" active="LIT7150Miracle" passive="EMML2066">
+                        <desc>
+                            <ref type="mss" corresp="EMML2066">EMML 2066, f. 60r, 115v</ref>
+                        </desc>
+                    </relation>
+                    <relation name="lawd:hasAttestation" active="LIT7150Miracle" passive="EMML2802">
+                        <desc>
+                            <ref type="mss" corresp="EMML2802">EMML 2803, f. 66v</ref>
+                        </desc>
+                    </relation>
+                    <relation name="lawd:hasAttestation" active="LIT7150Miracle" passive="EMML6196">
+                        <desc>
+                            <ref type="mss" corresp="EMML6196">EMML 6196, f. 19v</ref>
+                        </desc>
+                    </relation>
+                    <relation name="lawd:hasAttestation" active="LIT7150Miracle" passive="EMML7543">
+                        <desc>
+                            <ref type="mss" corresp="EMML7543">EMML 7543, f. 27r</ref>
+                        </desc>
```

Comment:
you MUST have relations to describe the connection between the miracles

part of
different from
version of

all these things must be stated in relations, and also probably in the abstract